week-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,62 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "https://i.pinimg.com/736x/1c/fb/4c/1cfb4c52eef989b843109877ff5f1047.jpg",
```

Comment:
Search hot-linking images in slack. And list reasons for why or why not should we use that.